zfs-prune-snapshots

```diff
@@ -6,7 +6,7 @@
 # Date: November 20, 2015
 # License: MIT
 
-VERSION='v1.5.0'
+VERSION='v1.5.1'
```

Comment:
Please bump the minor version here, as this PR changes the usage and arguments:

``` bash
VERSION='v1.6.0'
```

Reply:
ok, I will commit asap, best regards

Reply:
I even would reverse the version increment, and let you push a separate commit with the more eventual changes and the proper version upgrade you find useful